pkg/config/config.go

```diff
@@ -33,6 +33,7 @@ type Config struct {
 	Signers      SignerConfigs
 	Builder      BuilderConfig
 	Transparency TransparencyConfig
+	BuildType    BuildType
```

Comment:
We'll need to ensure that only a pre-defined set of values are allowed here.

Reply:
I let the respective `pipelineRun` and `taskRun` packages determine what values are acceptable. https://github.com/tektoncd/chains/pull/895/files#diff-46036042c702e6cd4cff49ca6f7eae50d0bc874b4b6e20da07eecca5df6a1581R104

Reply:
I think this is what you were talking about. https://github.com/tektoncd/chains/pull/895/files#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5R324